DigitalExcellence/dex-backend
# Bugfix/205 highlight toomuch info

## Description

HighlightResourceResult had a Project field which provided the frontend with too much information. Now it only sends projectId, projectName, highlightDescription and highlightId.

## Type of change


## Checklist


## Steps to Test or Reproduce

1. Clone frontend (related branch) and backend.
2. Run postman tests
3. Run frontend and check if highlights on the front page work.

## Link to issue

Closes: #205
